descriptors/ModuleDescriptor-template.json

```diff
@@ -582,6 +607,7 @@
             "scheduled-notice-storage.scheduled-notices.collection.get",
             "scheduled-notice-storage.scheduled-notices.item.delete",
             "scheduled-notice-storage.scheduled-notices.item.put",
+            "scheduled-notice-storage.scheduled-notices.item.post",
```

Comment:
Looked through the code and didn't find where this permission is used. How was missing of this dependency discovered?